test/local_testnet_upgrade_l1.sh

```diff
@@ -30,7 +30,7 @@ VALIDATORS_API_PORT_START=1317
 HEADS_IP_START=20
 CHAIN_ID="${CHAIN_ID:-devnet}"
 LOCALNET_DATADIR="$(pwd)/$CHAIN_ID"
-UPGRADE_VERSION="v0.4.0"
+UPGRADE_VERSION="v0.4.1"
```

Comment:
needs to be v0.5.0

Reply:
https://github.com/allora-network/allora-chain/pull/594